mypy/test/testpep561.py

```diff
@@ -32,27 +32,100 @@
 alpha_func(2)
 """
 
+C_EXT_PROGRAM = """
+from typedpkg_c_ext.foo import speak
+from typedpkg_c_ext.hello import helloworld
+
+speak("abc")
+speak(1)
+"""
+
+
+class NamespaceProgramImportStyle(Enum):
+    from_import = """\
+from typedpkg_nested.nested_package.nested_module import nested_func
```

Comment:
I dislike having those long module names. If you give each of them names of 3-5 letters it will become more readable (and you won't have to break long lines with `\` below). Also a comment may be in order explaining why some things must go together on one line (IIUC it's so that the line numbers in the error messages are consistent?)